WalletWasabi/WabiSabi/Crypto/CredentialIssuer.cs

```diff
@@ -99,17 +99,17 @@ public class CredentialIssuer
 		/// <param name="registrationRequest">The request containing the credentials presentations, credential requests and the proofs.</param>
 		/// <returns>The <see cref="CredentialsResponse">registration response</see> containing the issued credentials and the proofs.</returns>
 		/// <exception cref="WabiSabiCryptoException">Error code: <see cref="WabiSabiCryptoErrorCode">WabiSabiErrorCode</see></exception>
-		public CredentialsResponse HandleRequest(CredentialsRequest registrationRequest)
-			=> PrepareResponse(registrationRequest).Commit();
+		public Task<CredentialsResponse> HandleRequest(CredentialsRequest registrationRequest, CancellationToken cancel)
```

Comment:
@lontivero was right about this at the prev PR. This should be named `HandleRequestAsync`. It is confusing if we are awaiting a non-async named function and the compiler complaining that we should use the one with the `CancellationToken` because it thinks the two methods are identical.

Reply:
Yeah I was confused by that too. I will rename it